framework/Web/UI/WebControls/TPageLoadTime.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * TPageLoadTime class file.
+ *
+ * @author Brad Anderson <belisoful@icloud.com>
+ * @link https://github.com/pradosoft/prado
+ * @license https://github.com/pradosoft/prado/blob/master/LICENSE
+ * @package Prado\Web\UI\WebControls
+ */
+
+namespace Prado\Web\UI\WebControls;
+
+/**
+ * TPageLoadTime class.
+ *
+ * Writes the amount of time taken from application start to rendering the contents of this control.
+ *
+ * @author Brad Anderson <belisoful@icloud.com>
+ * @package Prado\Web\UI\WebControls
+ * @since 4.2.0
+ */
+class TPageLoadTime extends TLabel
```

Comment:
Do we really need a web control for this? TLogger already does the same, and its output can be routed to different places as needed

Reply:
There are webpages that link in the load time.  It was to make use of the application start time.